docs/deploy.md

```diff
@@ -0,0 +1,148 @@
+# Deploy Guide
+
+## Prerequisite Software
+
+The following software should be installed on the host before attempting to deploy Classy.
```

Comment:
probably want to add the versions on each line (at least the versions we're using now)